extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-fluid.js

```diff
@@ -202,7 +202,7 @@ describes.realWin('DoubleClick Fast Fetch Fluid', realWinConfig, env => {
         sandbox.spy(impl, 'connectFluidMessagingChannel');
     const onFluidResizeSpy = sandbox.spy(impl, 'onFluidResize_');
     impl.attemptChangeHeight = () => Promise.resolve();
-    return utf8Encode(rawCreative).then(creative => {
+    return Promise.resolve(utf8Encode(rawCreative)).then(creative => {
```

Comment:
We can remove this `#then`, making the inner code inline with the above line.

Reply:
done